### PR TITLE
Design review feedback

### PR DESCRIPTION
**Pull request by chris-lawton**
Address feedback that was covered in the 1st design review session. The spreadsheet is linked from the ticket containing the feedback.

Edit: The mobile search feedback will be covered in its own ticket as Ben was doing a mock up for it.

### CODE REVIEW

File: wagtailio/static/sass/components/_logo-card.scss [hunk 1]
```diff
@@ -34,9 +37,29 @@
         margin-top: auto;
     }
 
-    &:is(a):hover {
-        #{$root}__heading {
-            color: $color--teal;
+    &:is(a) {
```
Comment (stevedya) [inline wagtailio/static/sass/components/_logo-card.scss:40]: **Praise:** Love the use of the futuristic pseudo selector here. 

Robot swinging arms

**Review by stevedya — APPROVED** [review]
Looks great! Just some font size suggestions

File: wagtailio/static/sass/base/_typography.scss [hunk 2]
```diff
@@ -36,8 +36,8 @@ h1,
 
 h2,
 .heading-two {
-    font-size: 1.875rem;
-    line-height: 2.5125rem;
+    font-size: 1.7rem;
+    line-height: 2.1125rem;
```
[the remark this answers is not in the document]
Comment (chris-lawton) [reply wagtailio/static/sass/base/_typography.scss:40]: Ah yeah, thanks!

File: wagtailio/static/sass/components/_cta.scss [hunk 3]
```diff
@@ -18,7 +18,7 @@
 
     > svg {
         transition: transform 0.3s;
-        margin: 0 15px 0 auto;
+        margin: 0 0  0 auto;
```
[the remark this answers is not in the document]
Comment (chris-lawton) [reply wagtailio/static/sass/components/_cta.scss:29]: All good, i'd be suggesting the same : )

File: wagtailio/static/sass/components/_logo-card.scss (hunk 1 above)
Comment (chris-lawton) [reply wagtailio/static/sass/components/_logo-card.scss:40]: 😄 I can see myself using this a lot more going forward!

**Pull request merged**